src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -102,7 +102,7 @@ private JObject GetInvokeResult(byte[] script, Signer[] signers = null, Witness[
                     }
                     catch (Exception ex)
                     {
-                        stack.Add("error: " + ex);
+                        stack.Add("error: " + ex.Message);
```

Comment:
```suggestion
                        stack.Add(item.GetInterface<object>().GetType().Name);
```

Reply:
In this way, it will throw exception directly but can't get result stack!
![725922486f0147421b02d13395e875b](https://github.com/neo-project/neo-modules/assets/38493437/b1c99bd1-cd20-47ae-9475-e48b98858c27)
Compare to current code:
![302dc02b1c0fea24c7e69980f02256f](https://github.com/neo-project/neo-modules/assets/38493437/b6d8829d-2749-487b-9249-d614567acc35)